exampleSite/config.toml

```
@@ -53,7 +53,7 @@ disqusShortname = "yourdiscussshortname"
     custom_css = []
 
     # Custom JS
-    custom_js = []
+    custom_js = [""]
```

Just one thing now.
```suggestion
    custom_js = []
```
Leaving an empty string here causes this:
```
<script src="https://deploy-preview-358--hugo-coder.netlify.app/"></script>
```
An `script` entry for a non-existing JS file point to the base URL.

---

Well, I can commit to your branch, so I just did it.